packages/block-template/templates/react/tsconfig.json

```diff
@@ -3,7 +3,7 @@
     "allowJs": true,
     "allowSyntheticDefaultImports": true,
     "isolatedModules": true,
-    "jsx": "preserve",
+    "jsx": "react-jsx",
```

Comment:
As above RE typescript/JSX.

Reply:
Replied in https://github.com/blockprotocol/blockprotocol/pull/441#r925489980